docs/content/getting-started/deployment/via-clusterctl.md

```diff
@@ -12,7 +12,7 @@ key from this block below:
 ```yaml
 providers:
   - name: "caren"
-    url: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/latest/runtime-extension-components.yaml"
+    url: "https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/releases/v{{< param "version" >}}/runtime-extension-components.yaml"
```

Comment:
What is `{{< param "version" >}}`, is that something a user needs to set?

Reply:
No this will be rendered with the correct value when building and publishing the docs.

This is a Hugo shortcode (https://gohugo.io/content-management/shortcodes/) that I created previously (https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/docs/layouts/shortcodes/param.html) and already added to other pages (https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/docs/content/getting-started/create-cluster/aws.md?plain=1#L36) to dynamically inject params from Hugo config file (https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/docs/hugo.toml#L73) or specified at build time (https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/make/docs.mk#L12).

The version here is updated by release-please at release time (see release please config at https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/main/release-please-config.json#L11). The CAREN v0.9.0 release was the first time this release please functionality was used to update files as part of release and it worked really well!